dask/async.py

```diff
@@ -488,12 +488,10 @@ def fire_task():
                 task = dsk[key]
                 _execute_task(task, data)  # Re-execute locally
             else:
-                raise type(res)('\nRemote Exception:\n'
-                              + '-----------------\n'
-                              + str(res) + '\n\n'
-                              + 'Traceback:\n'
-                              + '----------\n'
-                              + tb)
+                print("Traceback:\n"
+                      "----------")
+                print(tb)
```

Comment:
Ah, I didn't think about catching print statements.  I was however thinking about catching normal exceptions.  It would be nice to be able to catch the underlying `UnicodeEncodeError`.  Perhaps a dynamically generated Exception class?